packages/modules/event-bus-local/src/services/event-bus-local.ts

```diff
@@ -112,7 +113,8 @@ export default class LocalEventBusService extends AbstractEventBusModuleService
   }
 
   async releaseGroupedEvents(eventGroupId: string) {
-    const groupedEvents = this.groupedEventsMap_.get(eventGroupId) || []
+    let groupedEvents = this.groupedEventsMap_.get(eventGroupId) || []
+    groupedEvents = JSON.parse(JSON.stringify(groupedEvents))
```

Comment:
Do you think this could have some major impact on block event loop when the grouped events will be huge? Like 200K ids or we will never reach that number

Reply:
No recommendation to be honest. I was thinking out loud to see if we need cloning of data. But, I think you would have thought about that already 👍